scripts/docker/gatkbase/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Using OpenJDK 17
 # This Dockerfile does not require any files that are in the GATK4 repo.
+# NOTE: If you updated the ubtuntu version make sure to update the samtools/bcftools/bedtools versions in the README
```

Comment:
ubtuntu -> ubuntu

Why are the genomics tools versions dependent on ubuntu version?

Reply:
Can you add a similar note about updating the README in the conda template yaml?

Reply:
It's been a while since I've used Ubuntu desktop but thought that certain versions set upper limits on the apt repo versions, so some packages latest versions won't work on older Ubuntu releases... is that right? If not then I'm not sure where the comment could go. The apt commands install whatever latest is available at the time, so it's possible if they aren't pinned to the distro version then they could change underneath us without realizing with a new build?

Reply:
So it looks like something like this is happening (and versions might've already changed under us without realizing!). I made a simple ubuntu docker from 20.04 and then from 24.04 and installed `samtools` using the exact same updating commands from our Dockerfile. The earlier version installed 1.10 and the latter installed 1.19.2. So I think the apt repos are somehow pinned to the ubuntu version, but it's not clear if those could still update with the same ubuntu point release.